pkg/system/stat_unix_test.go

```diff
@@ -31,7 +31,8 @@ func TestFromStatT(t *testing.T) {
 	if stat.Gid != s.GID() {
 		t.Fatal("got invalid gid")
 	}
-	if stat.Rdev != s.Rdev() {
+        //nolint:unconvert // conversion needed to fix mismatch types on mips64el
```

Comment:
Looks like this is still using spaces instead of tabs

Also, going forward, could you update your existing PRs instead of opening new ones with each change?

Reply:
I do not know it. Sorry!
I will modify the code to use tabs. I will Reopen PR to trigger the CI action.

Thank you !

Reply:
You have closed this PR a few days ago, but I think this update makes sense. 
I fixed and resubmitted your code, hope you don't mind. ref #40732